### PR TITLE
[13.0][MIG] website_sale_invoice_address: Migration to 13.0

### PR DESCRIPTION
**Pull request by chienandalu**
[MIG] website_sale_invoice_address: Migration to 13.0

cc @Tecnativa TT28630

ping @ernestotejeda @sergio-teruel

### CODE REVIEW

**Review by ernestotejeda — APPROVED** [review]

**Review by ernestotejeda — COMMENTED** [review]
Travis is red

File: website_sale_invoice_address/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fe--commerce-lightgray.png?logo=github
+    :target: https://github.com/OCA/e-commerce/tree/12.0/website_sale_invoice_address
+    :alt: OCA/e-commerce
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_invoice_address
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/113/12.0
```
Comment (sergio-teruel) [inline website_sale_invoice_address/README.rst]: regenerate readme

**Review by sergio-teruel — CHANGES_REQUESTED** [review]
Minor comment and waiting to fix travis

File: website_sale_invoice_address/README.rst (hunk 1 above)
Comment (chienandalu) [reply website_sale_invoice_address/README.rst]: changes done

**Comment by chienandalu** [comment]
green now

**Review by sergio-teruel — APPROVED** [review]
Functional test OK...

**Comment by pedrobaeza** [comment]
/ocabot migration website_sale_invoice_address
/ocabot merge nobump

Please remember to put a proper PR title

**Pull request merged**